Sources/SafeDICore/Models/Initializer.swift

```diff
@@ -236,12 +242,14 @@ public struct Initializer: Codable, Hashable {
                 innerLabel = node.firstName.text
             }
             typeDescription = node.type.typeDescription
+            hasDefaultValue = node.defaultValue != nil
```

Comment:
SwiftSyntax really coming through with this one! Extraordinarily simple to check if there is a default value in the code.